docs/Tutorials/Advanced-Tutorials/NPC-Images/Conversations.md

```diff
@@ -0,0 +1,92 @@
+---
+icon: material/message-processing
+tags:
+  - Conversation
+  - NPC Images
+---
+
+Since we have modified our resource pack with all our pixel art images, we can now proceed with updating our QuestPackage
+with all the necessary changes. 
+
+<div class="grid" markdown>
+!!! example "Related Docs"
+    * [Conversations](../../../Documentation/Features/Conversations.md)
+</div>
+
+## Changing the menu_conv_io
+
+Let's modify our QuestPackage to use a different conversation style. 
+Paste this code into your `package.yml`:
+
+``` YAML title="package.yml" linenums="1"
+    menu_conv_io:
+        npc_wrap: '&f              '
+        npc_text: '&f              {npc_text}'
+        npc_name_align: left
+        npc_name_format: '{npc_name}'
+        npc_name_newline_separator: true #Personal Preference
+        option_text: '&e              {option_text}'
+        option_selected: '&e              {option_text}'
+ 
+        control_select: jump,left_click 
+        control_cancel: sneak 
+```
+
+What does this do?
+
+This wraps the text a certain amount of spaces to the right, allowing some space for our new image. 
+This space might vary upon personal preference and can easily be extended by adding some more spaces.
+
+!!! danger
+    Changing this in one **QuestPackage** does change it globally!
```

Comment:
I think it worth mention, that this is currently a bad BQ design, and i would suggest to use something like a ConfigSetup package only containing this setup